README.md

```diff
@@ -39,6 +38,12 @@ gazelle(
 )
 ```
 
+### Using WORKSPACE
+
+The `gazelle_cc` is build and distributed using using Bazel modules. It is possible to use it using legacy WORKSPACE defintions, but it is neither tested nor encouraged.
```

Comment:
This is a little strong. Bazel is removing support for `WORKSPACE` files in 9.0 later this year, but we should still support `WORKSPACE` in at least 8.0. A lot of people will be stuck with `WORKSPACE` for a while.